bp_common/software/py/nbf.py

```diff
@@ -3,6 +3,21 @@
 #
 #   MEM (.mem) to Network Boot Format (.nbf)
 #
+#   NBF format is
+#   opcode_addr_data
+#   opcode map:
+#     8'h00: 1 byte write 
+#     8'h01: 2 byte write
+#     8'h02: 4 byte write
+#     8'h03: 8 byte write
+#
+#     8'h04: 1 byte read
+#     8'h05: 2 byte read
+#     8'h06: 4 byte read
+#     8'h07: 8 byte read
```

Comment:
Thoughts on having these be 8'h10 through 8'h13? Would provide nice symmetry with the write commands in that the size is still specified by the lower four bits as 2^N.

(this would also then align with what I selected for the ArtyParrot project for read commands, but it is very easy for us to change the choice of opcode, too)

Reply:
fine by me, since that still leaves room for fence and finish

Reply:
Cool. I think update these and then LGTM